[2023-12-20T11:42:15.733+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.second_id scheduled__2023-10-13T00:00:00+00:00 [queued]>
[2023-12-20T11:42:15.793+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.second_id scheduled__2023-10-13T00:00:00+00:00 [queued]>
[2023-12-20T11:42:15.793+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2023-12-20T11:42:15.843+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_id> on 2023-10-13 00:00:00+00:00
[2023-12-20T11:42:15.869+0000] {standard_task_runner.py:60} INFO - Started process 3947 to run task
[2023-12-20T11:42:15.887+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'second_id', 'scheduled__2023-10-13T00:00:00+00:00', '--job-id', '835', '--raw', '--subdir', 'DAGS_FOLDER/first_dat.py', '--cfg-path', '/tmp/tmpnx1r3bf1']
[2023-12-20T11:42:15.895+0000] {standard_task_runner.py:88} INFO - Job 835: Subtask second_id
[2023-12-20T11:42:16.039+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag.second_id scheduled__2023-10-13T00:00:00+00:00 [running]> on host 29b3723064ac
[2023-12-20T11:42:16.371+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abrham' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='second_id' AIRFLOW_CTX_EXECUTION_DATE='2023-10-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-13T00:00:00+00:00'
[2023-12-20T11:42:16.378+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-20T11:42:16.379+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '']
[2023-12-20T11:42:16.428+0000] {subprocess.py:86} INFO - Output:
[2023-12-20T11:42:16.432+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-20T11:42:16.596+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=second_id, execution_date=20231013T000000, start_date=20231220T114215, end_date=20231220T114216
[2023-12-20T11:42:16.718+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-20T11:42:16.766+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
